packages/adblocker-puppeteer/test/adblocker.test.ts

```diff
@@ -36,3 +37,34 @@ describe('#fromPuppeteerDetails', () => {
     });
   });
 });
+
+describe('#stylesInjection', () => {
+
+  it('does not inject styles into original content', async () => {
+    const browser = await puppeteer.launch();
+    try {
+      const page = await browser.newPage();
+      try {
+        const url = 'https://example.com';
```

Comment:
I guess it is not a big issue but in principle I would prefer to not have the tests depend on fetching some external resources. Maybe overkill but we could use a simple express server running on localhost scoped to this test, what do you think?

Reply:
Right you are! Now the tests are launching a local Express server and Puppeteer browser and page in the ```before``` hook, and closing them in the ```after``` hook. The blocker instance now just parses a single cosmetic rule without requesting the whole rules database.